packages/mask/src/plugins/Wallet/SNSAdaptor/SelectNftContractDialog.tsx

```diff
@@ -255,24 +255,20 @@ function ContractListItem(props: ContractListItemProps) {
     const { onSubmit, contract } = props
     const { classes } = useStyles()
     const chainId = useChainId()
+    const blockie = useBlockie(contract.contractDetailed.address)
 
     return (
         <div style={{ position: 'relative' }}>
             <ListItem className={classes.listItem} onClick={() => onSubmit(contract.contractDetailed)}>
-                {contract.contractDetailed.iconURL ? (
-                    <img className={classes.icon} src={contract.contractDetailed.iconURL} />
-                ) : null}
+                <img className={classes.icon} src={contract.contractDetailed.iconURL || blockie} />
```

Comment:
For `contract.contractDetailed.iconURL === ''`, we should use blockie.

`'' || 'abc' // return 'abc'`
`'' ?? 'abc' // return ''`